### PR TITLE
Server Info

### PR DESCRIPTION
**Pull request by weeryan17**
- [x] Completed PR - Has all features or fixes that it was meant to and isn't half done
- [x] Follows the style guide - Style guide found [here](https://github.com/FlareBot/FlareBot/blob/master/CONTRIBUTING.md)
- [x] Your code follows the same code style as the rest of the project
- [x] Tested changes

## What is added
Server info command

Examples:

### CODE REVIEW

File: src/main/java/stream/flarebot/flarebot/commands/general/ServerInfoCommand.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                "\n" +
+                "**Verification Level:** " +
+                getVerificationString(guild.getVerificationLevel()), true);
+        eb.setFooter("ID: " + guild.getId(), null);
+        eb.setColor(Color.BLUE);
+        channel.sendMessage(eb.build()).queue();
+    }
+
+    private String getVerificationString(Guild.VerificationLevel level){
+        switch (level){
```
Comment (weeryan17) [inline src/main/java/stream/flarebot/flarebot/commands/general/ServerInfoCommand.java]: Before anyone else comments on this. JDA doesn't really have a better way to do it that i found

Comment (ArsenArsen) [inline src/main/java/stream/flarebot/flarebot/commands/general/ServerInfoCommand.java]: Yes it did. The enums have a humane name associated with them.

Comment (weeryan17) [inline src/main/java/stream/flarebot/flarebot/commands/general/ServerInfoCommand.java]: I doesn't have the cool unicode associated with it for high and verry high

Comment (ArsenArsen) [inline src/main/java/stream/flarebot/flarebot/commands/general/ServerInfoCommand.java]: Oh, the PR got rejected. Sorry.

File: src/main/java/stream/flarebot/flarebot/commands/general/ServerInfoCommand.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                "**Roles:** " +
+                guild.getRoles().size() + "\n" +
+                "\n" +
+                "**Server region:** " +
+                guild.getRegion().getName() + "\n" +
+                "\n" +
+                "**Verification Level:** " +
+                getVerificationString(guild.getVerificationLevel()), true);
+        eb.setFooter("ID: " + guild.getId(), null);
+        eb.setColor(Color.BLUE);
```
Comment (binaryoverload) [inline src/main/java/stream/flarebot/flarebot/commands/general/ServerInfoCommand.java]: Color should be `CYAN` refer to pin in #contributors in future

File: src/main/java/stream/flarebot/flarebot/commands/general/ServerInfoCommand.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    }
+
+    private String getVerificationString(Guild.VerificationLevel level){
+        switch (level){
+            case HIGH:
+                return "(\u256F\u00B0\u25A1\u00B0\uFF09\u256F\uFE35 \u253B\u2501\u253B"; //(╯°□°）╯︵ ┻━┻
+            case VERY_HIGH:
+                return "\u253B\u2501\u253B\u5F61 \u30FD(\u0CA0\u76CA\u0CA0)\u30CE\u5F61\u253B\u2501\u253B"; //┻━┻彡 ヽ(ಠ益ಠ)ノ彡┻━┻
+            default:
+                return level.toString().charAt(0) + level.toString().substring(1).toLowerCase();
```
Comment (binaryoverload) [inline src/main/java/stream/flarebot/flarebot/commands/general/ServerInfoCommand.java]: You could change this to `WordUtils.capitalizeFully()` up to you

Comment (weeryan17) [inline src/main/java/stream/flarebot/flarebot/commands/general/ServerInfoCommand.java]: I mean normally it is fully caps

Comment (binaryoverload) [inline src/main/java/stream/flarebot/flarebot/commands/general/ServerInfoCommand.java]: The method I referenced does the exact same thing as you do here, the name is misleading I know

Comment (weeryan17) [inline src/main/java/stream/flarebot/flarebot/commands/general/ServerInfoCommand.java]: Not a big deal

File: src/main/java/stream/flarebot/flarebot/commands/general/ServerInfoCommand.java [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+                "\n" +
+                "**Voice**\n" +
+                "Total: " +
+                guild.getVoiceChannels().size() + "\n" +
+                "\n" +
+                "Active: " +
+                guild.getVoiceChannels().stream().filter(voiceChannel -> voiceChannel.getMembers().size() > 0).count() + "\n" +
+                "\n" + afk, true);
+        eb.addField("Misc info", "**Creation time:** " +
+                guild.getCreationTime().toLocalDateTime().atOffset(ZoneOffset.UTC).format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")) + " GMT/BST\n" +
```
Comment (WalshyDev) [inline src/main/java/stream/flarebot/flarebot/commands/general/ServerInfoCommand.java]: It'd be better to just say UTC

Comment (ArsenArsen) [inline src/main/java/stream/flarebot/flarebot/commands/general/ServerInfoCommand.java]: Timezones are a huge confusion. ~~Just ask Numnberphile or Computerphile, not sure which.~~

Use this formattter: `yyyy-MM-dd HH:mm:ss 'UTC'`

**Pull request merged**